### PR TITLE
support meter network for token price fetching

### PR DESCRIPTION
**Pull request by simonzg**
### What was wrong? 👾

Price fetching for tokens on meter network was not supported, including native tokens and asset token addresses.
The fallback price fetching seems not right with 
`self.get_token_usd_price(token_address) / self.get_ether_usd_price`

### How was it fixed? 🎯

For the native token price, register meter network in `get_token_eth_price_from_oracles` to fetch the MTR price with coingecko client.

For token address, there are no compatible oracles on meter network, so I configured the fallback price fetching with coingecko.

The fallback price should be `self.get_token_usd_price(token_address) / self.get_native_coin_usd_price()`

### CODE REVIEW

**Comment by simonzg** [comment]
I have read the CLA Document and I hereby sign the CLA

File: safe_transaction_service/tokens/clients/coingecko_client.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ class CoingeckoClient(BaseHTTPClient):
         EthereumNetwork.POLYGON: "polygon-pos",
         EthereumNetwork.POLYGON_ZKEVM: "polygon-zkevm",
         EthereumNetwork.CELO_MAINNET: "celo",
+        EthereumNetwork.METER_MAINNET: "meter",
+        EthereumNetwork.METER_TESTNET: "meter"
```
Comment (moisses89) [inline safe_transaction_service/tokens/clients/coingecko_client.py:37]: Could you remove testnet asset?

**Review by moisses89 — CHANGES_REQUESTED** [review]
Could you include your test here? https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/tokens/tests/test_price_service.py#L67C9-L67C39
Thanks for your contribution.

File: safe_transaction_service/tokens/clients/coingecko_client.py (hunk 1 above)
Comment (simonzg) [reply safe_transaction_service/tokens/clients/coingecko_client.py]: testnet is now removed

File: safe_transaction_service/tokens/services/price_service.py [hunk 2]
```diff
@@ -449,7 +454,7 @@ def get_token_eth_price_from_oracles(self, token_address: ChecksumAddress) -> fl
         """
         return (
             self.get_token_eth_value(token_address)
-            or self.get_token_usd_price(token_address) / self.get_ether_usd_price()
+            or self.get_token_usd_price(token_address) / self.get_native_coin_usd_price()
```
[the remark this answers is not in the document]
Comment (simonzg) [reply safe_transaction_service/tokens/services/price_service.py:457]: got it, thanks

**Comment by simonzg** [comment]
Added test case as requested.

**Review by moisses89 — APPROVED** [review]

**Comment by Uxio0** [comment]
@simonzg please fix linting

**Review by Uxio0 — CHANGES_REQUESTED** [review]
@simonzg please fix linting. Tests are not passing either

**Review by Uxio0 — APPROVED** [review]

**Comment by Uxio0** [comment]
Thanks for your PR!

**Pull request merged**